### PR TITLE
added initial .gitignore

### PR DESCRIPTION
**Pull request by Jalle19**

### CODE REVIEW

**Comment by ksooo** [comment]
Nothing to add.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# build artifacts
+build
+
+# clion project (auto-generated from CMakeLists.txt)
```
Comment (ksooo) [inline .gitignore]: clion project?

Comment (Jalle19) [inline .gitignore]: On my old Thinkpad I am.

Comment (ksooo) [inline .gitignore]: :-O

Comment (Jalle19) [inline .gitignore]: Can we still merge it? :D

Comment (ksooo) [inline .gitignore]: NO! ... Well, on second thought... YES. ;-)

**Pull request merged**